tests/snuba/api/endpoints/test_organization_group_index.py

```diff
@@ -1556,7 +1556,7 @@ def get_response(self, *args, **kwargs):
     @patch("sentry.api.helpers.group_index.eventstream")
     @patch("sentry.eventstream")
     def test_delete_by_id(self, mock_eventstream_task, mock_eventstream_api):
-        eventstream_state = object()
+        eventstream_state = {"event_stream_state": uuid4()}
```

Comment:
The object was changing after upgrading celery. I suspect that to more accurately simulate actually calling the task, they're serializing/deserializing when we use `CELERY_ALWAYS_EAGER=True`